tox.ini

```diff
@@ -51,7 +51,7 @@ commands =
     unit: python -m pytest test/unit/ --cov={toxinidir}/molecule/ --no-cov-on-fail {posargs}
     functional: python -m pytest test/functional/ {posargs}
     lint: python -m flake8
-    lint: python -m yamllint -s test/ molecule/
+    lint: python -m yamllint -s .
```

Comment:
This the the major goal of this change: so we avoid adding YAML files to other locations that do not have standardized formatting.
Also this makes much eaiser to run run yamllint outside tox or from other tools like IDEs, hooks,...